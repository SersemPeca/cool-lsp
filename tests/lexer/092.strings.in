"\0\1\2\3\\0\\1\\2\\3\n\t\b\f\a\b\c\d\e"
