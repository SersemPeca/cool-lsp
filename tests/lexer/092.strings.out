#1 STR_CONST "0123\\0\\1\\2\\3\n\t\bfa\bcde"
